support-files/mariadb.pc.in

```diff
@@ -20,4 +20,4 @@ Description: @CPACK_PACKAGE_DESCRIPTION_SUMMARY@
 URL: @CPACK_PACKAGE_URL@
 Version: @VERSION@
 Libs: -L${libdir} @LIBS_FOR_CLIENTS@
-Cflags: -I${includedir} -I${includedir}/.. @CFLAGS_FOR_CLIENTS@
```

Comment:
What does this line do?

Reply:
I removed the `-I${includedir}/..` line which is not necessary (and adding `/usr/include` to the explicit include line messed up `#include_next` for some stdlib headers).